manifests/operator.yaml

```diff
@@ -23,7 +23,7 @@ rules:
     resources:
       - securitycontextconstraints
     resourceNames:
-      - nonroot
+      - anyuid
```

Comment:
@alinadonisa we may need to start the init container using the operator SA, I don't want the db SA to have anyuid